lib/server/connect.js

```diff
@@ -100,6 +178,29 @@ Meteor.publish('kadira.debug.listeners', function() {
   });
 });
 
+function getAppEnv() {
```

Comment:
Move all these to a new file called `utils.js` and create a new global namespace `Utils`. And assign these functions to it.